palindrome.c

```diff
@@ -0,0 +1,40 @@
+//Albert Zhang
+//Checks if a string is a palindrome but I can't get it to work please help me
+#include <stdio.h>
+#include <string.h>
+int palindrome(char i[], char j[], int count) {
```

Comment:
also, when you submit code that doesn't work you should submit with plenty of comments!